src/view/com/composer/state/composer.ts

```diff
@@ -304,9 +308,13 @@ export function createComposerState({
   }
   let quote: Link | undefined
   if (initQuoteUri) {
-    quote = {
-      type: 'link',
-      uri: initQuoteUri,
+    // TODO: Consider passing the app url directly.
```

Comment:
Seems fine as is, but leaving a note here in case you wanted to remember to come back here.

Reply:
this one actually seems like it's a bit more convenient to leave as is, we're relying on `initQuote.uri` being an `at://` uri in a different place